ArduCopter/config.h

```diff
@@ -326,6 +326,14 @@
 
 //////////////////////////////////////////////////////////////////////////////
 
+//////////////////////////////////////////////////////////////////////////////
+// Weathervane - allow vehicle to yaw into wind
+#ifndef WEATHERVANE_ENABLED
+# define WEATHERVANE_ENABLED !HAL_MINIMIZE_FEATURES
```

Comment:
Is this what we are usually using instead?

```suggestion
# define WEATHERVANE_ENABLED BOARD_FLASH_SIZE <= 1024
```

Reply:
Have we changed how we are doing this?  I though that the point of HAL_MINIMISE_FEATURES was exactly this:
https://github.com/ArduPilot/ardupilot/blob/427c08db2674ad4e02a7907a173b75a7f92bf5a8/libraries/AP_HAL/AP_HAL_Boards.h#L180

Reply:
Well we are trying to avoid adding more of those.  There are only like 9 boards `HAL_MINIMIZE_FEATURES 1` . So in the end them main defining feature to not compile something in is available flash.